src/GovUk.Education.ExploreEducationStatistics.Common/Extensions/MigrationBuilderExtensions.cs

```diff
@@ -41,5 +45,10 @@ public static class MigrationBuilderExtensions
                 migrationBuilder.Sql(line);
             }
         }
+
+        private static string GetEnvironment() =>
+            Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") ??
+            Environment.GetEnvironmentVariable("DOTNET_ENVIRONMENT") ??
+            Environments.Production;
```

Comment:
Interesting to know - does Azure set this to production by default for the other APIs? 🤔 I assume so since the boxes are pre-configured to run dotnet but just curious

Reply:
Hey @luke-h1,

Yes it does and that's kind of the reason why I did it like this.

Here's a few references to it behaving this way in the docs:

https://learn.microsoft.com/en-us/aspnet/core/fundamentals/host/web-host?view=aspnetcore-8.0#environment

![image](https://github.com/user-attachments/assets/430a53da-8569-4c9f-9093-d3d49776b05e)

https://learn.microsoft.com/en-us/aspnet/core/fundamentals/environments?view=aspnetcore-8.0#azure-app-service

![image](https://github.com/user-attachments/assets/7a0db874-2d7b-4e22-8991-9b71b39ef0cf)

https://learn.microsoft.com/en-us/aspnet/core/fundamentals/environments?view=aspnetcore-8.0#environments

![image](https://github.com/user-attachments/assets/3e10a057-a17b-4c4a-a49c-4e3d4d062c5f)